### PR TITLE
Parse the NODE_OPTS environment variable in the Node runner

### PR DESCRIPTION
**Pull request by s-leroux**
Add the content of the `NODE_OPTS` environment variable to the NodeJS' arguments in the node runner. This is notably useful to enable remote debugging while testing:

 ```
NODE_OPTS=--inspect-brk=0.0.0.0:9993 opal -Itest -Ilib -Ivendored-minitest \
                                           -soptparse -sio/console -stimeout -srubygems -stempfile -smonitor \
                                           -Rnodejs tmp/minitest_node_nodejs.rb
```

The string in `NODE_OPTS` is parsed using `Shellwords` (available in Ruby ≥ 1.8 according to [this SO answer](https://stackoverflow.com/a/6068282/2363712))

### CODE REVIEW

File: lib/opal/cli_runners/nodejs.rb [hunk 1]
```diff
@@ -14,10 +15,13 @@ def self.call(data)
         argv = data[:argv].dup.to_a
         argv.unshift('--') if argv.any?
 
+        opts = Shellwords.shellwords(ENV['NODE_OPTS'] || "")
```
Comment (elia) [inline lib/opal/cli_runners/nodejs.rb:18]: nit: the linter is failing on this double quotes, can you please fix them?

**Review by elia — APPROVED** [review]
Thanks! 👍👍👍 
(Can be merged after the linter is back to green)

File: lib/opal/cli_runners/nodejs.rb (hunk 1 above)
Comment (s-leroux) [reply lib/opal/cli_runners/nodejs.rb]: > nit: the linter is failing on this double quotes, can you please fix them?

Fixed

**Comment by s-leroux** [comment]
Could someone check why `build/minitest` has failed on the CI? As far as I can tell, it doesn't seem related to this PR.

**Comment by hmdne** [comment]
It's a random failure, Chromium didn't start on time. I'm rerunning the CI.

**Pull request merged**